subsystems/motor/ServoSubsystem.java

```diff
@@ -0,0 +1,242 @@
+package org.usfirst.frc4904.standard.subsystems.motor;
+
+
+import org.usfirst.frc4904.standard.commands.Idle;
+import edu.wpi.first.wpilibj.Servo;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class ServoSubsystem extends Subsystem {
+	protected final Servo[] servos;
+	protected boolean isInverted;
+	protected double lastPosition;
+	// Constants from wpilib's Servo.java.
```

Comment:
`wpilib.Servo.kMaxServoAngle` is private (only reason this constant exists)
It starts with `k` because I just copied these from Servo. The name change is a good idea though.